editor/src/components/editor/editor-component.tsx

```diff
@@ -88,6 +90,8 @@ function useDelayedValueHook(inputValue: boolean, delayMs: number): boolean {
 }
 
 export const EditorComponentInner = React.memo((props: EditorProps) => {
+  const room = useRoom()
+  console.info('room', JSON.stringify(room.getStatus()))
```

Comment:
probably worth removing these, but I guess they'll be chomped soon anyways if not

Reply:
Yeah, I thought it would be useful to keep them so we can see it "working" in other environments until people put some real code in here.